testbed/main.cc

```diff
@@ -83,18 +89,18 @@ bool random_distribution = false;
 
 GLFWwindow* window = NULL;
 
-template <class C> void FreeClear( C & cntr ) {
-    for ( typename C::iterator it = cntr.begin();
-              it != cntr.end(); ++it ) {
-        delete * it;
-    }
-    cntr.clear();
+template <class C>
+void FreeClear( C & cntr ) {
+  for (typename C::iterator it = cntr.begin();  it != cntr.end(); ++it) {
```

Comment:
Why this change?

Reply:
Homogenize the style in this file.
- That method used 4 space indentation.
- The line break in the middle of the for() is unwarranted, The line isn't long.
- Line break after template declation